src/components/05_pages/Content/Content.js

```diff
@@ -80,221 +80,286 @@ class Content extends Component {
     includes: PropTypes.shape({
       'user--user': PropTypes.object,
     }),
+    links: PropTypes.shape({
+      next: PropTypes.string,
+    }),
   };
   static defaultProps = {
     contentList: [],
     includes: {},
+    links: {},
   };
   state = {
     contentTypes: [],
+    scrollToTop: false,
     status: null,
     sort: { path: 'changed', direction: 'DESC' },
+    page: {
+      offset: 0,
+      limit: 50,
+    },
   };
   componentDidMount() {
     this.props.requestContentTypes();
     this.props.requestContent(this.state);
   }
+  componentDidUpdate(prevProps) {
+    if (
+      this.state.scrollToTop &&
+      JSON.stringify(prevProps.contentList) !==
+        JSON.stringify(this.props.contentList)
+    ) {
+      window.scrollTo(0, this.table.offsetTop);
+    }
+  }
   tableSortHandler = (path, direction) => () => {
-    this.setState({ sort: { path, direction } }, () => {
-      this.props.requestContent(this.state);
-    });
+    this.setState(
+      {
+        sort: { path, direction },
+        page: { offset: 0, limit: this.state.page.limit },
```

Comment:
I guess it is fine to jump to the first page when sorting. For now at least it is totally fine, and its rather a UX team decision how it should behave.